### PR TITLE
Changes to the README, mostly on the CDS data store

### PR DESCRIPTION
**Pull request by sjboeing**

### CODE REVIEW

**Comment by sjboeing** [comment]
Hi @leifdenby: I have added Ralph's suggested changes to the README here. Let me know if it needs a changelog (it is only changes to the README, not the code itself).

**Comment by sjboeing** [comment]
Addresses #193.

File: README.md [hunk 1]
```diff
@@ -131,13 +134,16 @@ Or use one of the domain definitions included with `lagtraj` (e.g.
...
 ```
-the `<start_date>` and `<end_date>` should be formatted as `YYYY/MM/DD`, e.g. `2020/02/01` for the 2nd of February 2020.
+the `<start_date>` and `<end_date>` should be formatted as `YYYY/MM/DD`, e.g. `2020/02/02` for the 2nd of February 2020.
 
 An optional flag `--retry-rate <num_minutes>` causes `lagtraj` to continue
 retrying download of submitted data requests every `num_minutes` minutes until
 all data has been downloaded. Every time this command is run it will attempt to
 download only data not yet downloaded.
 
+You can monitor the status of your requests via the [CDS requests page](ihttps://cds.climate.copernicus.eu).
```
Comment (leifdenby) [inline README.md:144]: I think you have a superfluous `i` at the beginning of the link, should be `https://` I think

**Review by leifdenby — CHANGES_REQUESTED** [review]
Thanks for doing this. Apart for a small typo this looks good to me :)

**Comment by sjboeing** [comment]
Thanks @leifdenby: happens to me every once in a while with vi!

**Comment by leifdenby** [comment]
Looks good to merge. I would update the changelog too, keeps a record of the work you've done :) Also, there was a typo in the readme before. You've made the readme better, so that's an "improvement" to add to the changelog

**Pull request merged**